tests/modules/NotificationTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ *
+ * NotificationTest.php is unit test file for Notification module (modules/Notification.php).
```

Comment:
You can add a test to see if the account_sid, auth_token and from_number are getting set in the constructor as expected